### PR TITLE
enableUiEnhancements button

### PR DESCRIPTION
**Pull request by Laserlicht**
Since I added some features using this flag it would be good to have it on UI. To enable classic experience.

### CODE REVIEW

File: Mods/vcmi/config/vcmi/english.json [hunk 1]
```diff
@@ -90,6 +90,8 @@
 	"vcmi.systemOptions.framerateButton.help"   : "{Show FPS}\n\nToggle the visibility of the Frames Per Second counter in the corner of the game window.",
 	"vcmi.systemOptions.hapticFeedbackButton.hover"  : "Haptic feedback",
 	"vcmi.systemOptions.hapticFeedbackButton.help"   : "{Haptic feedback}\n\nToggle the haptic feedback on touch inputs.",
+	"vcmi.systemOptions.enableUiEnhancementsButton.hover"  : "Interface Enhancements",
+	"vcmi.systemOptions.enableUiEnhancementsButton.help"   : "{Interface Enhancements}\n\nToggle various quality of life interface improvements.",
```
Comment (Nordsoft91) [inline Mods/vcmi/config/vcmi/english.json:94]: Not very informative. Maybe enumerate at least most important points?

Comment (Laserlicht) [reply Mods/vcmi/config/vcmi/english.json:94]: This is intentional, because it would have to be constantly maintained. Should rather be in docs I think.

Comment (Nordsoft91) [reply Mods/vcmi/config/vcmi/english.json:94]: I assuming option is enabled by default? If so, you can add something like you have in PR description: "... disable this option if you want classic experience from original game"

Comment (IvanSavenko) [reply Mods/vcmi/config/vcmi/english.json:94]: Possibly describe 2-3 most noticeable changes,e.g. "such as large spell book, spell icons for scrolls, etc"?

And maybe add more complete version somewhere in the docs?

**Comment by Nordsoft91** [comment]
New button overlaps with "Town screen" label

Снимок экрана 2023-09-26 в 20 20 45

**Comment by Laserlicht** [comment]
ah... tested it on android... there is space. :)

Do we have space at the bottom? To move everything one line down?

**Comment by Nordsoft91** [comment]
I think so, but horizontal line probably needs to be adjusted
Снимок экрана 2023-09-26 в 20 40 42

**Comment by Nordsoft91** [comment]
Another option: replace 2 fullscreen toggles with one list/pop-up (same as for resolution selection) with items: windowed, fullscreen (borderless), fullscreen (exclusive)

**Comment by Laserlicht** [comment]
Is fine now. Moved the bottom part to the bottom

**Pull request merged**